src/lightning/store/__init__.py

```diff
@@ -1,3 +1,3 @@
-from lightning.store.cloud_api import download_from_cloud, load_model, upload_to_cloud
+from lightning.store.cloud_api import download_from_cloud, load_model, upload_model
 
-__all__ = ["download_from_cloud", "load_model", "upload_to_cloud"]
+__all__ = ["download_from_cloud", "load_model", "upload_model"]
```

Comment:
My comment here overlaps with Adrian's above comment but should it be named like this?
```suggestion
__all__ = ["download_model", "load_model", "upload_model"]
```

Reply:
done :)